draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,395 @@
+# Hybrid Deferred Dev Fund ZIP
+
+    ZIP: Unassigned {numbers are assigned by ZIP editors}
+    Title: Hybrid Deferred Development Fund
+    Owners: Jason McGee <aquietinvestor@gmail.com>
+            Skylar Saveland <skylar@free2z.com>
+            Peacemonger < >
+            Gguy <babygguy@gmail.com
```

Comment:
```suggestion
            Gguy <babygguy@gmail.com>
```

Reply:
@NerdGGuy - You are now listed as an owner.